system/services/presideObjects/SqlSchemaSynchronizer.cfc

```diff
@@ -156,6 +161,10 @@ component {
 		,          struct relationships = {}
 
 	) {
+		var versions       = _getVersionsOfDatabaseObjects( [arguments.dsn] );
```

Comment:
It would be great if we didn't have to get all this data from the DB just in case we're going to _skip_ a column.

Is there a way where we could refactor this so that for fields doing `dbsync=false` that we don't need to have all the definition SQLs for the column?

Reply:
Hi @DominicWatson ,

I've done this changes please check latest [commit](https://github.com/pixl8/Preside-CMS/pull/900/commits/1a93a4da1c22efcbdb8ffddb69297e14478effc9)